src/utils/proposals.ts

```diff
@@ -5,6 +5,36 @@ import {
 } from './index';
 import moment from 'moment';
 
+export const isExpired = (proposal: Proposal): boolean => {
+  return (
+    proposal.stateInVotingMachine === VotingMachineProposalState.ExpiredInQueue
+  );
+};
+
+export const isBoosted = (proposal: Proposal, vote: Vote): boolean => {

```

Comment:
this can be voteBeforeBoosted and only return is the vote was done before boosted, thats what we care about.
You can also remove the notBoosted constant, not needed since you already have a boolean value on boosted